io.github.lpub3d.LPub3D.json

```diff
@@ -0,0 +1,183 @@
+{
+  "id": "io.github.lpub3d.LPub3D",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15",
+  "sdk": "org.kde.Sdk",
+  "command": "lpub3d24",
+  "rename-icon": "lpub3d",
+  "rename-desktop-file": "lpub3d.desktop",
+  "rename-appdata-file": "io.github.lpub3d.appdata.xml",
+  "build-options": {
+    "env": {
+      "LP3D_3RD_PARTY_DIR": "/stage/lpub3d_linux_3rdparty",
+      "LP3D_DIST_DIR_PATH": "/stage/lpub3d_linux_3rdparty",
+      "LDRAWDIR": "/stage/lpub3d_linux_3rdparty/ldraw",
+      "PLATFORM_CODE": "osl",
+      "TARGET_CPU": "$(uname -m)",
+      "PLATFORM_VER": "$(. /etc/os-release 2>/dev/null; [ -n \"$VERSION_ID\" ] && echo $VERSION_ID || true)"
+    }
+  },
+  "cleanup": [
+    "/stage"
+  ],
+  "finish-args": [
+    "--device=dri",
+    "--share=ipc",
+    "--socket=x11",
+    "--filesystem=home",
+    "--env=TMPDIR=/var/tmp",
+    "--env=QT_ENABLE_HIGHDPI_SCALING=1"
+  ],
+  "modules": [
+    {
+      "name": "part-archives",
+      "buildsystem": "simple",
+      "build-commands": [
+        "unzip -d ${LP3D_3RD_PARTY_DIR} -q /stage/lpub3d_linux_3rdparty/complete.zip",
```

Comment:
As I indicated earlier, `/` is read only. 

I'm not sure I understand what this staging is for.